### ISSUE TITLE
check_translations.py: Make it more obvious if something is a warning vs. an error.

### ISSUE THREAD

**Issue by pocmo** [mozilla-mobile/focus-android#1434]
From the output it's often hard to see if something is an error or a warning. A warning is something we can investigate asynchronously by opening a bug and then continue. However errors break the build and therefore are more important and need to be fixed.

**Comment by pocmo** [comment]
Currently we can only look at the "exit code" but this doesn't help us if there are multiple warnings/errors.

**Comment by devophile** [comment]
Hi @pocmo This is my first time on open source. I would like to contribute to this project. Can you help me in getting the structure of the project... I mean the way things are working so that I can work on this? Or any sources you would like to refer me to understand this. Thank you. :smile:

**Comment by pocmo** [comment]
@devophile Sure! This one is for developers who want to write some Python code. There's a script in our repository that runs whenever we import strings. It makes sure that all the placeholders we use in our strings.xml file are also used in the translated strings:
https://github.com/mozilla-mobile/focus-android/blob/master/tools/l10n/check_translations.py

Some checks are treated as error and others as warnings (errors return exit code 1):
https://github.com/mozilla-mobile/focus-android/blob/master/tools/l10n/check_translations.py#L75-L83

All those checks print a message but from the message you can't see if they are an error or a warning: https://github.com/mozilla-mobile/focus-android/blob/master/tools/l10n/check_translations.py#L24-L43

To solve this issue it might be enough to just print "warning: " or "error: " in front of the messages. :)

**Comment by mastizada** [comment]
```
Checking for placeholders in translation files...
 * ERROR: [values-az/strings.xml] ...
 WARNING: [values-eo/strings.xml] ...
 WARNING: [values-pl/strings.xml] ...
```

**Comment by arjunkrishnababu96** [comment]
can I work on it?

**Comment by emilyeverett** [comment]
Hi, @pocmo, I see that this issue hasn't been assigned yet, and I'm interested in working on it. Mind if I take it on?

**Comment by pocmo** [comment]
Fixed by 1296ce9d55856893e9850ec72edfe1c435f47b21
